src/server/types.ts

```diff
@@ -204,7 +229,8 @@ export abstract class BaseDocumentDriveServer {
     abstract addDrive(drive: DriveInput): Promise<DocumentDriveDocument>;
     abstract addRemoteDrive(
         url: string,
-        options: RemoteDriveOptions
+        options: RemoteDriveOptions,
+        driveInfo?: DriveInfo
```

Comment:
Let's add the driveInfo to the RemoteDriveOptions, we could call it expectedDriveInfo?